addons/point_of_sale/static/src/js/models.js

```diff
@@ -1706,8 +1706,11 @@ exports.Orderline = Backbone.Model.extend({
     },
     _compute_all: function(tax, base_amount, quantity) {
         if (tax.amount_type === 'fixed') {
-            var sign_base_amount = base_amount >= 0 ? 1 : -1;
-            return (Math.abs(tax.amount) * sign_base_amount) * quantity;
+            var sign_base_amount = Math.sign(base_amount) || 1;
```

Comment:
@kebeclibre Why don't we just tax.amount * quantity?

Reply:
hmmmm good point. My answer is three fold:
- I wanted to have as little diff as possible, in code and logically
- We want that function to match the one in python, formally and logically
- Getting the sign of base_amount allows to handle having either the price or the quantity in negative (or both). Admittedly I don't think it is possible in the POS to have negative prices, but I wouldn't bet on it

Reply:
as discussed, mayb check if it should not be sign * abs(quantity) * tax.amound (no abs on tax.amount)

Reply:
@pimodoo you were right about the irrelevance of abs(tax_amount)

changes made :D